ieducar/intranet/include/clsCampos.inc.php

```diff
@@ -1764,8 +1764,8 @@ classe = (classe == \'formmdtd\')? \'formlttd tr_\' + This.nome : \'formmdtd tr_
 
                         $retorno .= "<tr><td colspan='2' class='$classe'><span class='form'><b>$componente[3]</b></span>{$explicacao}</td></tr>\n";
                     } else {
-                        if (isset($componente[10]) && ($componente[0] == 'textoDuploInv'
-                                || $componente[0] == 'textoInv')) {
+                        if (!empty($componente[10]) && !empty($componente[0])
```

Comment:
O comportamento foi modificado, `isset` retorna `true` para zero, string vazia ou false. É a intenção esta alteração de comportamento? 

Senão pode ser trocado por:

```php
if (isset($componente[10], $componente[0]) && ($componente[0] == 'textoDuploInv' || $componente[0] == 'textoInv')) {
    // code..
}
```

Reply:
Pelo que comrpeendi, o conteúdo da variável não é numérico e não deve ser vazio. Quando for vazio cai no else colocando a variável `$name` como vazia.